.github/workflows/scripts/script.sh

```diff
@@ -147,6 +146,18 @@ if [[ "$TEST" == "upgrade" ]]; then
   echo "Restarting in 60 seconds"
   sleep 60
 
+  # Let's reinstall pulpcore so we can ensure we have the correct dependencies
+  pwd
+  ls -al
+
+  cd ../pulpcore
+  git checkout -f ci_upgrade_test
+  pip install --upgrade --force-reinstall . ../pulp-cli ../pulp-smash
```

Comment:
pulp-cli is same level as pulpcore:
https://github.com/pulp/pulp_file/runs/6179363765?check_suite_focus=true#step:12:1520
```suggestion
  pip install --upgrade --force-reinstall ./pulpcore ./pulp-cli ./pulp-smash
```